libraries/index.html

```diff
@@ -11,62 +11,87 @@ <h1>CircuitPython Libraries</h1>
+        They also load faster, and for some low-RAM boards, are required because the <b>.py</b> files
+        are too large to compile on the board itself.
+        <strong>Unless you need the source version, choose the appropiate <b>.mpy</b> library bundle!</strong>
+        If you need to compile <b>.py</b> files to <b>.mpy</b>, you can
+        <a href="https://learn.adafruit.com/welcome-to-circuitpython/frequently-asked-questions#how-can-i-create-my-own-mpy-files-3020687-11">run the <b>mpy-cross</b>
+        cross-compiler yourself</a>.
       </p>
       <p>
         <b>To install</b>, download the appropriate bundle for your version of CircuitPython.
-        Unzip the file, open the resulting folder and find the lib folder. Open the lib
-        folder and find the library files you need to load. Create a lib folder on your
-        CIRCUITPY drive. Copy the individual library files you need to the lib folder
-        on your CIRCUITPY drive.
+        Unzip the file, open the resulting folder and find the <b>lib</b> folder. Open the lib
+        folder and find the library files you need to load. Create a <b>lib</b> folder on your
+        CIRCUITPY drive. Copy the individual library files you need to the <b>lib</b> folder
+        on your CIRCUITPY drive. More details are available
+        <a href="https://learn.adafruit.com/welcome-to-circuitpython/circuitpython-libraries">here</a>.
       </p>
       <p>
         You can always find the
         <a href="https://github.com/adafruit/Adafruit_CircuitPython_Bundle/releases/latest">latest releases of the libraries bundle</a>
         on GitHub.
+      </p>
+      <p>
+        An alternative way to download and update libraries on your board is to use the
+        <a href="https://learn.adafruit.com/keep-your-circuitpython-libraries-on-devices-up-to-date-with-circup">CircUp tool</a>,
+        which is under development.
+      </p>
+    </section>
+    <section>
+      <h2>Bundles</h2>
       <div class="release-section">
         <div id="adafruit-circuitpython-bundle-6.x-mpy">
-          <h3>Bundle Version 6.x</h3>
+          <h3>Bundle for Version 6.x</h3>
           <p>
             This bundle is built for use with CircuitPython 6.x.x. If you are using
             CircuitPython 6, please download this bundle.
           </p>
         </div>
         <div id="adafruit-circuitpython-bundle-7.x-mpy">
-            <h3>Bundle Version 7.x</h3>
+            <h3>Bundle for Version 7.x</h3>
             <p>
                 This bundle is built for use with CircuitPython 7.x.x. If you are using
-                CircuitPython 7, please download this bundle.
+                CircuitPython 7, please download this bundle. The .mpy format has
+                changed for CircuitPython 7; 6.x .mpy files are not compatible.
             </p>
         </div>
         <div id="adafruit-circuitpython-bundle-py">
-          <h3>Bundle Version py</h3>
+          <h3>Python Source Bundle</h3>
```

Comment:
Thank you for this - There have been multiple complaints about it not being called "source" making it impossible to find.